chia/farmer/farmer_api.py

```diff
@@ -321,13 +326,13 @@ async def respond_signatures(self, response: harvester_protocol.RespondSignature
         assert pospace is not None
         include_taproot: bool = pospace.pool_contract_puzzle_hash is not None
 
-        # we skip plot filter validation here, we trust the harvester
         computed_quality_string = verify_and_get_quality_string(
```

Comment:
can you add an assert in `verify_and_get_quality_string()` to ensure we always pass in either `height` or `filter_prefix_bits` too?